abrams_strogatz/lang_comp_bilinguals.py

```diff
@@ -0,0 +1,300 @@
+import matplotlib as mpl
```

Comment:
1:   1 D100 Missing docstring in public module [flake8-docstrings]
```suggestion
"""This module implements the Minett-Wang bilingual model."""
import matplotlib as mpl
```